Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:8-jre
+
+# sbt
+ENV SBT_URL=https://dl.bintray.com/sbt/native-packages/sbt
+ENV SBT_VERSION 0.13.15
+ENV INSTALL_DIR /usr/local
+ENV SBT_HOME /usr/local/sbt
+ENV PATH ${PATH}:${SBT_HOME}/bin
+ENV REDIS_IP=${REDIS_IP}
```

Comment:
how will take the ip REDIS ? 
suppose to be a command but everything is commented ?